packages/tx/src/baseTransaction.ts

```diff
@@ -91,8 +91,20 @@ export abstract class BaseTransaction<TransactionObject> {
     const rB = toBuffer(r === '' ? '0x' : r)
     const sB = toBuffer(s === '' ? '0x' : s)
 
-    this.nonce = new BN(toBuffer(nonce === '' ? '0x' : nonce))
-    this.gasLimit = new BN(toBuffer(gasLimit === '' ? '0x' : gasLimit))
+    const bufferedNonce = toBuffer(nonce === '' ? '0x' : nonce)
```

Comment:
if you want to use the same naming as above this can be `nonceB`

Reply:
I guess that should have been obvious to me.  Thanks!